jest.config.ts

```diff
@@ -10,4 +10,5 @@ module.exports = {
   setupFiles: ["jest-webextension-mock"],
   testEnvironment: "./jest.custom-test-environment.ts",
   setupFilesAfterEnv: ["./jest.setup.ts"],
+  testTimeout: 20000,
```

Comment:
Is this (still) needed?  
Or this needed for CI only? If yes, can this be smaller? Otherwise it might take forever till the pipeline eventually fails.